charts/langkit/values.yaml

```diff
@@ -3,17 +3,26 @@ replicaCount: 3
 nameOverride: ""
 fullnameOverride: ""
 
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
 ingress:
   enabled: false
   className: ""
   annotations: {}
-    # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
```

Comment:
any reason this is commented out?

Reply:
the ingress resource is disabled by default. The annotations are implementation specific to their ingress controller or gateway. These were meant to be examples rather than fixed suggestions.